### ISSUE TITLE
[Regression] Small physics bug in LittleBigPlanet [NPUA80472] (#10809)

### ISSUE THREAD

**Issue by kylometra** [RPCS3/rpcs3#12377]
### Quick summary

The wall the Skulldozer knocks down in the "Skulldozer" level falls over early.

### Details

There isn't a whole lot I can say besides what's said in the summary, but here are some videos:

Regression:
https://user-images.githubusercontent.com/30480654/180102847-c1d63fa3-5ddb-4c38-96c2-9480aa9fbb2c.mp4

Previous behaviour:
https://user-images.githubusercontent.com/30480654/180103126-0faa0bb0-3cde-4352-a27d-14087faf49c5.mp4

### Build with regression

v0.0.18-12773-9e62ca56 https://github.com/RPCS3/rpcs3/pull/10809

### Attach two log files

https://drive.google.com/file/d/1RDmLgIKd5QfYcMm2fkTwe5Jk-wD9bI9-/view?usp=sharing

### System configuration

Windows 10.0.19044 Build 19044
Intel(R) Core(TM) i7-9700K CPU @ 3.60GHz
NVIDIA GeForce GTX 1660 Ti Driver Version 27.21.14.6089

**Comment by TorutheRedFox** [comment]
joints seem to have regressed too in both LBP1 and 2 (especially pistons, they act like strings now)

**Comment by AniLeo** [comment]
@Nekotekina any idea? Very weird as it works with approximate but not accurate
Also, can someone retest just to make sure it's still an issue after all the fixes that happened in between?

**Comment by TorutheRedFox** [comment]
yeah it's weird

accurate xfloat made my pod just completely break one time too

**Comment by ruvaldak** [comment]
> @Nekotekina any idea? Very weird as it works with approximate but not accurate Also, can someone retest just to make sure it's still an issue after all the fixes that happened in between?

Yes this is still an issue. - in fact, it may be even worse, as it's occurring on approximate xfloat now as well.

**Comment by Pacomatic1** [comment]
Some other clips:

Accurate XFloat: 
https://github.com/user-attachments/assets/44361a71-e61a-4db6-982a-227770f2bb47
Approximate XFloat: 
https://github.com/user-attachments/assets/7e94d42a-1a01-4c50-bcd2-b792207940ce

Level: Skate to Victory
Approximate is the intended behavior.

**Comment by AniLeo** [comment]
Please retest, there were some changes to CPU accuracy not too long ago

**Comment by Pacomatic1** [comment]
Accurate: https://github.com/user-attachments/assets/0f076175-473f-49cc-aca9-9324d2d84f76
Approximate: https://github.com/user-attachments/assets/2357bc22-3dfa-4de2-8278-4c22139f3091

Using Accurate XFloat, everything seems to be fine expect the piston and the rod.
When they aren't stiff, they act like strings. For the rod, this is fine. For the piston, this means that it doesn't push and pull like it should.
When they are stiff, we get the same bug from before.

Approximate XFloat is still intended behaviour.

**Comment by digant73** [comment]
@kylometra is that still an issue?

**Comment by Jonathan44062** [comment]
This should be closed? I mean, I can complete the level without any issues, however, the door animation seems to be odd, on the PS3 Foogate, it should be closed before opening 

RPCS3
https://drive.google.com/file/d/1La13RTi9XR4wq1dd8lATYJS7snKSUoSJ/view?usp=sharing

PS3
https://youtu.be/CNUoAVoGljI?t=2663

[RPCS3.log.gz](https://github.com/user-attachments/files/27071719/RPCS3.log.gz)

**Comment by kylometra** [comment]
@Jonathan44062 Sure, you can still clear the level, but the issue described is still present as you say
